packages/react/src/components/Alert/Alert.tsx

```diff
@@ -46,7 +46,7 @@ export interface AlertProps
    */
   accessibility?: Accessibility
 
-  /** An Alert can contain action buttons. */
+  /** An alert can contain action buttons. */
```

Comment:
We should standardize on capitalization conventions across all components, but for now I think it's sufficient to make them at least consistent within a single component.